support/scripts/gcov/genhtml_report.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
```

Comment:
For any scripts that are 3rd party work imported, please add information that allows complete traceability in your commit message, i.e. filename, repository, branch, commit.